### ISSUE TITLE
Wrong behaviour sonata-project/admin-bundle version 3.10.1

### ISSUE THREAD

**Issue by aserratorta** [sonata-project/SonataAdminBundle#4233]
#### Sonata packages

```bash
$ composer show sonata-project/*
"sonata-project/admin-bundle": "3.10.1",
```

#### PHP version

```bash
$ php -v
PHP 7.0.8 (cli) (built: Jun 26 2016 12:30:44) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.8, Copyright (c) 1999-2016, by Zend Technologies
    with Xdebug v2.4.0RC3, Copyright (c) 2002-2015, by Derick Rethans

```

## Subject
I upgraded vendors yesterday. It removed sonata-project/admin-bundle version 3.10.0 and instaled 3.10.1. I show you the desconfiguration of some templates below.

 Expected results. version 3.10.0

![image](https://cloud.githubusercontent.com/assets/12971681/21203738/2aeb2b60-c254-11e6-9330-a51e84303ee8.png)

![image](https://cloud.githubusercontent.com/assets/12971681/21203757/4e323ba4-c254-11e6-9ab8-cf95aab67468.png)

## Actual results. version 3.10.1

![image](https://cloud.githubusercontent.com/assets/12971681/21203810/9f5efc38-c254-11e6-8bef-aede3800b8bb.png)

![image](https://cloud.githubusercontent.com/assets/12971681/21203785/7829657c-c254-11e6-9aba-dc98c75cf09e.png)

**Comment by StudioMaX** [comment]
This was fixed in https://github.com/sonata-project/SonataAdminBundle/pull/4232, wait when it will be tagged.

**Comment by davidromani** [comment]
👍

**Comment by jordisala1991** [comment]
Fixed in 3.10.2

**Comment by aserratorta** [comment]
Good!!
